integration/tests/dump_logs/dump_logs_test.go

```diff
@@ -0,0 +1,60 @@
+//go:build integration
+// +build integration
+
+package dump_logs
+
+import (
+	"os"
+	"testing"
+
+	"github.com/pkg/errors"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/weaveworks/eksctl/integration/tests"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/testutils"
+)
+
+var params *tests.Params
+
+func init() {
+	testing.Init()
+	if err := api.Register(); err != nil {
+		panic(errors.Wrap(err, "unexpected error registering API scheme"))
+	}
+	params = tests.NewParams("dump_logs")
+}
+
+func TestDumpLogs(t *testing.T) {
```

Comment:
Please extract this into a suite file.

Reply:
Could you please show me an example of what you mean? I am not quite sure I understand what a "suite" file is in this context. Thanks!

Reply:
ah sorry, completely missed this ping. :/ In all other folders look for a file ending `suit_test.go`. :) We use that to set up the suit.

Reply:
I am still having trouble finding these `suite_test.go` files you mentioned. I am looking under the `integrations/tests` directory. None of the sub-directories of this path have a `suite_test.go` file as far as I can see. If you could please explain the purpose of this file, I can try to implement it. Thanks!

Reply:
Ah, never mind, this is fine. :)